skaffold.yaml

```diff
@@ -6,7 +6,7 @@ build:
       custom:
         buildCommand: |
           sbt core/assembly keytool/assembly && \
-            docker build $BUILD_CONTEXT --file kubernetes/l0.Dockerfile --tag $IMAGE --build-arg BUILD_VERSION=$(cut -d \" -f 2 version.sbt) && \
+            docker build $BUILD_CONTEXT --file kubernetes/l0.Dockerfile --tag $IMAGE --build-arg BUILD_VERSION=$(cut -d \" -f 2 version.sbt) --platform linux/amd64 && \
```

Comment:
shouldn't it be configurable by the user? not everyone is running it on amd64 🤔

Reply:
That's the point. You can run it on `arm` but it will get build for `x86-64` anyway.